debian/rules

```diff
@@ -1,4 +1,13 @@
 #!/usr/bin/make -f
 export DH_VERBOSE=1
+export PYBUILD_AFTER_INSTALL=rm -rf '{destdir}/{install_dir}/examples'
 %:
-	dh $@ --with python2 --buildsystem=python_distutils
+	dh $@ --with python2 --buildsystem=pybuild --test-nose
+
+override_dh_installman:
+	make -C docs man
+	dh_installman
+	rm docs/source/for-ops/commands/*
+	rm docs/source/for-ops/commands.rst
```

Comment:
because they are generated by sphinx. If they are not removed, dpkg-buildpackage then fail due to source directory not being the same as the orig.tgz content. 

But I agree it would be cleaner to add a filter in the debian/options. I'll fix my PR ASAP